litmus-portal/frontend/src/containers/layouts/Scaffold/styles.ts

```diff
@@ -12,14 +12,14 @@ const useStyles = makeStyles((theme: Theme) => ({
     gridTemplateAreas: '"header header" "sidebar content"',
 
     '& ::-webkit-scrollbar': {
-      width: '0.4rem',
+      width: '0.35rem',
     },
     '& ::-webkit-scrollbar-track': {
-      marginTop: theme.spacing(1),
       webkitBoxShadow: `inset 0 0 8px ${theme.palette.common.black}`,
```

Comment:
This was added to avoid the scrollbar merging into the header as the color of the header has changed now, scrolling all the way to the top makes it so that the scrollbar merges into the header

Reply:
Yes i checked this but with the white color on top it looks like we have more area to scroll up to, may be a scroll bar track color can solve the problem if this is the case. @vanshBhatia-A4k9

Reply:
Its also overriding all the litmus ui graph legend table scrollbars, modal/pop over scrollbars and yaml editor scrollbar